### PR TITLE
doc: fix step numbers

### PR DESCRIPTION
**Pull request by amyreye**
And, seeing if I can do a pull request.

### CODE REVIEW

File: doc/getting-started/getting-started.rst [hunk 1]
```diff
@@ -361,7 +361,7 @@ Generate a Board Configuration File
 
    a. Make sure the USB disk is connected to the target.
 
-   a. Copy ``my_board.xml`` to the USB disk:
+   #. Copy ``my_board.xml`` to the USB disk:
```
Comment (dbkinder) [inline doc/getting-started/getting-started.rst:364]: good catch :)

File: doc/getting-started/getting-started.rst [hunk 2]
```diff
@@ -466,7 +466,7 @@ To generate a scenario configuration file and launch script:
       .. image:: ./images/gsg_config_launch_load.png
          :class: drop-shadow
 
-      #. Click the **Generate Launch Script** button.
+   #. Click the **Generate Launch Script** button.
```
Comment (dbkinder) [inline doc/getting-started/getting-started.rst:469]: ditto

**Review by dbkinder — APPROVED** [review]
LGTM, thanks for catching these.

**Comment by amyreye** [comment]
@dbkinder Thanks! Should I merge my own pull request?

**Pull request merged**